groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/lanthanides/RareEarthProcessing.groovy

```diff
@@ -0,0 +1,1046 @@
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+CENTRIFUGE.recipeBuilder()
+        .fluidInputs(fluid('thfree_monazite_leach') * 1000)
+        .fluidInputs(fluid('ammonia_solution') * 1000)
+        .outputs(metaitem('dustMonaReHydroxides'))
+        .fluidOutputs(fluid('refree_monazite_leach') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+CENTRIFUGE.recipeBuilder()
+        .fluidInputs(fluid('refree_monazite_leach') * 1000)
+        .fluidInputs(fluid('ammonia_solution') * 1000)
+        .outputs(metaitem('dustUranylHydroxidePrecipitate') * 1000)
+        .fluidOutputs(fluid('monazite_waste') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+// Xenotime Pre-processing
+
+for (fuel in sintering_fuels) {
+    if (!fuel.isPlasma) {
+        ROTARY_KILN.recipeBuilder()
+                .inputs(ore('dustSodaAsh') * 24)
+                .inputs(ore('dustXenotime'))
+                .fluidInputs(fluid(fuel.name) * fuel.amountRequired)
+                .fluidInputs(fluid('oxygen') * 50)
+                .outputs(metaitem('dustRoastedXenotime'))
+                .fluidOutputs(fluid('carbon_dioxide'))
+                .duration(400)
+                .EUt(Globals.voltAmps[2])
+                .buildAndRegister()
+    }
+}
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustRoastedXenotime'))
+        .fluidInputs(fluid('distilled_water') * 4000)
+        .fluidOutputs(fluid('xenotime_slurry') * 4000)
+        .duration(100)
+        .EUt(30)
+        .buildAndRegister()
+
+VACUUM_CHAMBER.recipeBuilder()
+        .notConsumable(metaitem('item_filter'))
+        .fluidInputs(fluid('xenotime_slurry') * 4000)
+        .outputs(metaitem('dustXenotimeResidue'))
+        .fluidOutputs(fluid('xenotime_phosphate_waste') * 4000)
+        .duration(160)
+        .EUt(80)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustXenotimeResidue'))
+        .fluidInputs(fluid('nitric_acid') * 1000)
+        .fluidOutputs(fluid('crude_xenotime_nitrate_solution') * 1000)
+        .duration(80)
+        .EUt(120)
+        .buildAndRegister()
+
+VACUUM_CHAMBER.recipeBuilder()
+        .notConsumable(metaitem('item_filter'))
+        .fluidInputs(fluid('crude_xenotime_nitrate_solution') * 1000)
+        .chancedOutput(metaitem('dustXenotimeTailing'), 1000, 0)
+        .fluidOutputs(fluid('xeno_re_nitrates_solution') * 1000)
+        .duration(100)
+        .EUt(80)
+        .buildAndRegister()
+
+SIFTER.recipeBuilder()
+        .inputs(ore('dustXenotimeTailing'))
+        .chancedOutput(metaitem('dustCassiterite') * 3, 500, 0)
+        .chancedOutput(metaitem('dustThoriumDioxide') * 3, 500, 0)
+        .chancedOutput(metaitem('dustUraniumDioxide') * 3, 500, 0)
+        .duration(100)
+        .EUt(30)
+        .buildAndRegister()
+
+// Bastnasite Concentrate Processing
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustBastReHydroxides'))
```

Comment:
This step and the next one can be combined (and also remove the bast_re_nitrates_solution)